README.md

```diff
@@ -281,6 +287,30 @@ The Client ID and the Client Secret will be needed for the basic script you will
    
 You can now restart your Flexy to restart the Java application. Messages will be sent to your OCS endpoint.
 
+## JVM Logger
+The OSIsoft connector makes use of the [HMS Solution Center logging library](https://github.com/hms-networks/sc-flexy-logger-lib) to ensure all logging follows HMS standards for general connectors.
+
+To add a different logging level in your configuration file, specify the logging level under the AppConfig object.
```

Comment:
saying add a different implies there is a default. Should the default level (if one is not specified in the config) be documented here?

Reply:
It should not log at all if non is given, so I will add a issue to make 4 the default logging level later. Going to reword this